core/SettingsPiwik.php

```diff
@@ -213,6 +213,15 @@ public static function getPiwikUrl()
         return $url;
     }
 
+    /**
+     * @see SettingsPiwik::isPiwikInstalled
+     * @return bool
+     */
+    public static function isMatomoInstalled()
+    {
+        return self::isPiwikInstalled();
+    }
```

Comment:
Should isPiwikInstalled() be deprecated?

Reply:
good point 👍  will do